_pages/resources.html

```diff
@@ -2,6 +2,7 @@
 layout: default
 title: Resources
 permalink: /resources
+description: Here are some great resources for any other FRC teams
```

Comment:
How about "Resources for other FRC teams and presentation logs from our preseason conventions."